kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/occultism/shaped.js

```diff
@@ -37,6 +37,15 @@ onEvent('recipes', (event) => {
                 C: '#forge:inlays/pewter'
             },
             id: 'occultism:crafting/sacrificial_bowl'
+        },
+        {
+            output: 'occultism:magic_lamp_empty',
+            pattern: [' A ', 'ABA', ' AA'],
+            key: {
+                A: '#forge:ingots/silver',
+                B: 'atum:horus_godshard'
```

Comment:
Isn't there a tag for specific godshards?

Reply:
unfortunately, no. There's a genereal `atum:godshards` but nothing for individual shards